api/lib/domain/services/mail-service.js

```diff
@@ -348,155 +255,80 @@ function sendCertificationCenterInvitationEmail({
   code,
   locale,
 }) {
-  let templateParams, fromName, subject;
-  const frenchFranceTemplateParams = {
-    certificationCenterName,
-    pixHomeName: PIX_HOME_NAME_FRENCH_FRANCE,
-    pixHomeUrl: PIX_HOME_URL_FRENCH_FRANCE,
-    pixCertifHomeUrl: `${config.domain.pixCertif + config.domain.tldFr}`,
-    redirectionUrl: `${
-      config.domain.pixCertif + config.domain.tldFr
-    }/rejoindre?invitationId=${certificationCenterInvitationId}&code=${code}`,
-    supportUrl: HELPDESK_FRENCH_FRANCE,
-    ...frTranslations['certification-center-invitation-email'].params,
-  };
-  const frenchSpokenTemplateParams = {
-    certificationCenterName,
-    pixHomeName: PIX_HOME_NAME_INTERNATIONAL,
-    pixHomeUrl: `${config.domain.pix + config.domain.tldOrg}`,
-    pixCertifHomeUrl: `${config.domain.pixCertif + config.domain.tldOrg}`,
-    redirectionUrl: `${
-      config.domain.pixCertif + config.domain.tldOrg
-    }/rejoindre?invitationId=${certificationCenterInvitationId}&code=${code}`,
-    supportUrl: HELPDESK_FRENCH_SPOKEN,
-    ...frTranslations['certification-center-invitation-email'].params,
-  };
-  const englishSpokenTemplateParams = {
+  locale = locale ?? FRENCH_FRANCE;
+  const mailerConfig = _getMailerConfig(locale);
+
+  const subject = mailerConfig.translation['certification-center-invitation-email'].subject;
+  const fromName = mailerConfig.translation['email-sender-name']['pix-certif'];
+  const templateVariables = {
     certificationCenterName,
-    pixHomeName: PIX_HOME_NAME_INTERNATIONAL,
-    pixHomeUrl: PIX_HOME_URL_INTERNATIONAL_ENGLISH_SPOKEN,
-    pixCertifHomeUrl: `${config.domain.pixCertif + config.domain.tldOrg}?lang=en`,
-    redirectionUrl: `${
-      config.domain.pixCertif + config.domain.tldOrg
-    }/rejoindre?invitationId=${certificationCenterInvitationId}&code=${code}&lang=en`,
-    supportUrl: HELPDESK_ENGLISH_SPOKEN,
-    ...enTranslations['certification-center-invitation-email'].params,
+    pixHomeName: mailerConfig.homeName,
+    pixHomeUrl: mailerConfig.homeUrl,
+    pixCertifHomeUrl: mailerConfig.pixCertifHomeUrl + (locale === FRENCH_FRANCE ? '' : `?lang=${locale}`),
```

Comment:
Suggestion: 
Faire une fonction `_formatUrl` qui ajoute le paramètre lang à la fin de l'url dans le cas de la locale FRENCH_FRANCE pour éviter ce ternaire un peu partout dans le fichier.

Quelque chose comme ça : 

```
function _formatUrl(url, locale) {
  const formattedURL = new URL(url);

  if(locale === FRENCH_FRANCE) {
    formattedURL.searchParams.set("lang", locale);
  }

  return url.toString();
}
```

qui serait ensuite utiliser dans ce cas : 
```suggestion
    pixCertifHomeUrl: _formatUrl(mailerConfig.pixCertifHomeUrl),
```